### DISCUSSION TITLE
Is there any plan to support LocalStorage for browsers?

### DISCUSSION THREAD

**Discussion by alphashr** [humanwhocodes/humanfs#18] [Ideas]
Great work! 

Just a quick question, is there any plan to support LocalStorage to be able to use this in Browser?

Based on the fsx architecture, it could be a new _impl_ if I am not wrong.

**Comment by nzakas** [comment]
It could be. My plans right now are focused on cloud storage solutions next, but I could definitely circle back and look at localStorage.

**Comment by atjn** [comment]
I think a more appropriate storage solution for the web would be the [origin private file system](https://developer.mozilla.org/en-US/docs/Web/API/File_System_API/Origin_private_file_system), but you could also implement both.

**Comment by alphashr** [comment]
Thanks @nzakas  for implementing the [opfs runtime](https://github.com/humanwhocodes/humanfs/pull/49) 🎉

**Reply by nzakas** [reply]
You're welcome! Kept forgetting to come back and comment here. :smile:
